Annex60/Fluid/Examples/Performance/Example1v1.mo

```diff
@@ -71,10 +71,10 @@ For this script to work, make sure that Dymola stores at least 4 results.
 cpuOld=OutputCPUtime;<br/>
 evaluateOld=Evaluate;<br/>
 OutputCPUtime:=true;<br/>
-simulateModel(\"Annex60.Fluid.Examples.Performance.Example1v1(allowFlowReversal.k=true, from_dp.k=false)\", stopTime=10000, numberOfIntervals=10, method=\"dassl\", resultFile=\"Example1v1\");<br/>
-simulateModel(\"Annex60.Fluid.Examples.Performance.Example1v2(from_dp.k=true, allowFlowReversal.k=true)\", stopTime=10000, numberOfIntervals=10, method=\"dassl\", resultFile=\"Example1v2\");<br/>
-simulateModel(\"Annex60.Fluid.Examples.Performance.Example1v1(allowFlowReversal.k=false, from_dp.k=false)\", stopTime=10000, numberOfIntervals=10, method=\"dassl\", resultFile=\"Example1v1\");<br/>
-simulateModel(\"Annex60.Fluid.Examples.Performance.Example1v1(allowFlowReversal.k=false, from_dp.k=true)\", stopTime=10000, numberOfIntervals=10, method=\"dassl\", resultFile=\"Example1v1\");<br/>
+simulateModel(\"Annex60.Fluid.Examples.Performance.Example1v1(allowFlowReversal.k=true, from_dp.k=false)\", stopTime=10000, Interval=1000, method=\"dassl\", resultFile=\"Example1v1\");<br/>
```

Comment:
These changes should be reverted as these lines are the Dymola-specific scripts. They have a different syntax than the MSL, and Dymola uses `numberOfIntervals` in its scripting language.

The same applies to the other examples.